net/rs/client-gen-dotnet/tests/snapshots/generator__basic_works.snap

```diff
@@ -0,0 +1,63 @@
+---
+source: rs/client-gen-dotnet/tests/generator.rs
+expression: "gen(idl, \"Basic\")"
+---
+using global::Sails.Remoting;
+using global::Sails.Remoting.Abstractions;
+using global::Sails.Remoting.Abstractions.Core;
+using global::System;
+using global::System.Collections.Generic;
+
+#pragma warning disable RCS0056 // A line is too long
```

Comment:
Do we need this? Ananlyzers shouldn't check generated code

Reply:
There are might be plenty of other ananlyzers enable which you might not be aware of - it passes this one, but fails on the other :-)